windows_containers/index.adoc

```diff
@@ -0,0 +1,33 @@
+[id="windows-container-overview"]
+= Windows container overview
+include::modules/common-attributes.adoc[]
+:context: windows-container-overview
+
+toc::[]
+
+Windows Container Support for {product-title} is a feature providing the ability to run Windows compute nodes in an {product-title} cluster. This is possible by using the Red Hat Windows Machine Config Operator (WMCO) to install and manage Windows nodes. By using Red Hat subscription, you can get support for Windows Container. For more information see, xref:../windows_containers/windows-containers-release-notes-4-x.adoc#windows-containers-release-notes-4-x[Release notes].
+
+xref:../windows_containers/understanding-windows-container-workloads.adoc#understanding-windows-container-workloads[Windows container workloads]:
+For workloads including both Linux and Windows, {product-title} allows you to deploy Windows workloads running on Windows Server containers while also providing traditional Linux workloads hosted on Red Hat Enterprise Linux CoreOS (RHCOS) or Red Hat Enterprise Linux (RHEL).
+Windows container workloads support for clusters running on the following cloud providers:
+
+* Amazon Web Services (AWS)
+* Microsoft Azure
```

Comment:
This list will differ for each targeted version of OCP. These were the initial supported platforms, but WMCO is now also supported for vSphere and bare metal. See the [prereqs section](https://docs.openshift.com/container-platform/4.9/windows_containers/understanding-windows-container-workloads.html#supported-platforms-based-on-openshift-container-platform-and-wmco-versions) for more info.

Reply:
acked. Removed the listing